Automattic/jetpack
# Update/optimized calypso prettier package

- Run prettier before any changes (formats 1 unpretty file)
- Update to an optimized bundle of our prettier flavor that provides the executable without any excess. Details: https://github.com/Automattic/wp-calypso/pull/29987
- Update the lockfile (`yarn`)
- Run prettier again (no change/noop)


#### Testing instructions:

* Checkout
* `yarn` to update the dependencies
* `yarn run reformat-files` should leave the repository in a clean state (prettier is the same before and after)

#### Proposed changelog entry for your changes:

* None
